docs/general/ad-filtering/create-own-filters.md

```diff
@@ -4045,31 +4053,50 @@ Exception rules can disable some scriptlets on particular domains. The syntax fo
   if not set, all scriptlets will not be applied;
 - `arguments` — optional, a list of `string` arguments to match the same blocking rule and disable it.
 
-For example, if you want to disable the rule
+**Examples**
 
-```adblock
-example.org,example.com#%#//scriptlet("abort-on-property-read", "alert")
-```
+1. Disable specific scriptlet rule so only `abort-on-property-read` will be applied
+   only on `example.org` and its subdomains:
 
-for the `example.com` domain, the following exception rules could be used:
+    ```adblock
+    example.org,example.com#%#//scriptlet("abort-on-property-read", "alert")
+    example.com#@%#//scriptlet("abort-on-property-read", "alert")
+    ```
 
-- to disable a particular scriptlet only:
+1. Disable all `abort-on-property-read` scriptlets for `example.com` and its subdomains:
 
-```adblock
-example.com#@%#//scriptlet("abort-on-property-read", "alert")
-```
+    ```adblock
+    example.org,example.com#%#//scriptlet("abort-on-property-read", "alert")
+    example.com#@%#//scriptlet("abort-on-property-read")
+    ```
 
-- to disable all `abort-on-property-read` scriptlets for `example.com`:
+1. Disable all scriptlets for `example.com` and its subdomains:
```

Comment:
how is it different from this?

1. Disable all `abort-on-property-read` scriptlets for `example.com` and its subdomains:

    ```adblock
    example.org,example.com#%#//scriptlet("abort-on-property-read", "alert")
    example.com#@%#//scriptlet("abort-on-property-read")
    ```

Reply:
`#@%#//scriptlet("abort-on-property-read")` disable all (and only) _abort-on-property-read_ scriptlets, so other scriptlet won't be affected

`#@%#//scriptlet()` disables any scriptlet rule